lib/chord-metadata/docker-compose.yml

```diff
@@ -57,6 +57,8 @@ services:
       #interval: 30s
       #timeout: 20s
       #retries: 3
+    volumes:
+      - shared-data:/shared
```

Comment:
This will work but the issue is that using a shared-data volume across multiple containers may result in file system i/o issue. Especially if you have different containers using the same file path with the shared volume

Reply:
I think this is a temporary state: for now, it's possible that this will cause a conflict between htsget and candig-server, but as I switch this to minio, that shouldn't be a problem. Otherwise, this is primarily a staging ground for ingest data and won't be used again after the ingest.